other/upgrade_2-1_postgresql.sql

```diff
@@ -1,6 +1,37 @@
 /* ATTENTION: You don't need to run or use this file! The upgrade.php script does everything for you! */
 
 /******************************************************************************/
+--- Fixing dates...
+/******************************************************************************/
+--# Updating old values
+UPDATE {$db_prefix}calendar
+SET start_date = '1001-01-01'
+WHERE start_date < '1001-01-01';
+
+UPDATE {$db_prefix}calendar
+SET end_date = '1001-01-01'
+WHERE end_date < '1001-01-01';
+
+UPDATE {$db_prefix}calendar_holidays
+SET event_date = '1001-01-01'
+WHERE event_date < '1001-01-01';
+
+UPDATE {$db_prefix}log_spider_stats
+SET stat_date = '1001-01-01'
+WHERE stat_date < '1001-01-01';
+
+UPDATE {$db_prefix}members
+SET birthdate = '1001-01-01'
+WHERE birthdate < '1001-01-01';
+
+--# Changing default values
+ALTER TABLE {$db_prefix}calendar CHANGE start_date start_date date NOT NULL DEFAULT '1001-01-01';
```

Comment:
wrong syntax for pg:
ALTER TABLE {$db_prefix}calendar ALTER COLUMN start_date SET DEFAULT '100-01-01'::date;